streams/src/main/java/org/apache/kafka/streams/KafkaStreaming.java

```diff
@@ -68,12 +71,12 @@
     private static final int STOPPED = 2;
     private int state = CREATED;
 
-    private final StreamThread[] threads;
+    private final List<StreamThread> threads;
```

Comment:
The article is talking about the use of array as an interface. Here we are using the array as an internal structure.